src/wp-includes/blocks.php

```diff
@@ -670,9 +670,11 @@ function has_blocks( $post = null ) {
  *
  * This test optimizes for performance rather than strict accuracy, detecting
  * whether the block type exists but not validating its structure and not checking
- * reusable blocks. For strict accuracy, you should use the block parser on post content.
+ * patterns (previously referred to as reusable blocks). For strict accuracy, you
+ * should use the block parser on post content.
  *
  * @since 5.0.0
+ * @since 6.3.0 Reusable Blocks renamed to Patterns.
```

Comment:
I don't think this is needed as none of the code has been renamed. The `@since` here is to denote major changes that impact usage of the function.
```suggestion
```